src/main/java/duke/DukeOutOfBoundsException.java

```diff
@@ -0,0 +1,4 @@
+package duke;
+
+public class DukeOutOfBoundsException extends Exception{
+}
```

Comment:
Consider adding constructors for your Exception classes.

Reply:
Okay, will do.